include/targets/cuda/random_helper.h

```diff
@@ -0,0 +1,108 @@
+#pragma once
+
+#include <curand_kernel.h>
+
+namespace quda {
+
+#if defined(XORWOW)
```

Comment:
Is there any reason we have this path? Is it just for reference (as something to enable for debugging reasons), or can we get rid of it?